pkg/apis/spinnaker/interfaces/interfaces.go

```diff
@@ -59,12 +64,22 @@ type SpinnakerService interface {
 	DeepCopySpinnakerService() SpinnakerService
 }
 
+// +kubebuilder:object:generate=false
 type SpinnakerServiceList interface {
 	runtime.Object
 	GetItems() []SpinnakerService
 	DeepCopySpinnakerServiceList() SpinnakerServiceList
+	GetContinue() string
+	GetRemainingItemCount() *int64
+	GetResourceVersion() string
+	GetSelfLink() string
+	SetContinue(c string)
+	SetRemainingItemCount(c *int64)
+	SetResourceVersion(c string)
+	SetSelfLink(c string)
```

Comment:
Do we still need these?

Reply:
We need it for the library `sigs.k8s.io/controller-runtime v0.11.2`